docs/src/index.md

```diff
@@ -1,17 +1,13 @@
 # EcoSISTEM.jl
 
 **EcoSISTEM** is a [Julia](http://www.julialang.org) package that provides functionality
-for simulating species undergoing dynamic biological processes such as birth, death,
-competition and dispersal, as well as environmental changes in climate and habitat. This
-repo has now been adapted to include basic functionality for spatially explicit, dynamic,
-stochastic SIR and other compartmental models.
+for simulating species undergoing dynamic biological processes such as birth, death, competition and dispersal, as well as environmental changes in climate and habitat.
 
-The package was primarily developed for global scale simulations of plant biodiversity.
-The underlying model for this is described in the arXiv paper [arXiv:1911.12257 (q-bio.QM)](https://arxiv.org/abs/1911.12257)
-*Dynamic virtual ecosystems as a tool for detecting large-scale responses of biodiversity
-to environmental and land-use change*. Future updates to the package functionality involve
-incorporating age-structure and more complex epidemiological models.
+The package was primarily developed for global scale simulations of plant biodiversity. The underlying model for this is described in the arXiv paper [arXiv:1911.12257 (q-bio.QM)](https://arxiv.org/abs/1911.12257)
+*Dynamic virtual ecosystems as a tool for detecting large-scale responses of biodiversity to environmental and land-use change*.
 
-This package is in alpha now, so please raise an issue if you find any problems.
+There are substantial changes to the package introduced through the [`dev`](https://boydorr.github.io/EcoSISTEM.jl/dev/) branch, including epidemiological simulations and refactoring of the code base for further flexibility.
+
+This package is in beta now, so please raise an issue if you find any problems. For more information on how to contribute, please read [our contributing guidelines](../../CONTRIBUTING.md).
```

Comment:
I don't think that link will work unfortunately - you'll need the path to the github site I think. You might also want to move all of these links down to the bottom of the page too like in the README.md.